backend/common/utils/utils.py

```diff
@@ -65,7 +65,7 @@ def path_join(base, *urls):
     return btpl._replace(path=path).geturl()
 
 
-class Float32JSONEncoder(json.JSONEncoder):
+class FloatJSONEncoder(json.JSONEncoder):
```

Comment:
Minor: I suppose it's not really just a float encoder if it handles ints :)

Reply:
true, but the entire thing only exists to handle NaN/Inf (which is floats). Otherwise, we would just skip this...